lib/broadside/deploy/ecs_deploy.rb

```diff
@@ -73,8 +76,16 @@ def bootstrap
         info "Service '#{family}' doesn't exist, creating..."
         EcsManager.create_service(config.ecs.cluster, family, @deploy_config.service_config)
       end
+    end
 
-      @deploy_config.bootstrap_commands.each { |command| run_command(command) }
+    def run_bootstrap_commands(update_task_def: false)
+      update_task_revision if update_task_def
```

Comment:
`update_task_def: false` is an example of the new Ruby keyword argument syntax.

Yeah, always updating the task def makes sense... it's not like we need to be worried about creating too many of them I guess. I just updated the code (removed the gate).